src/rpc/handlers/AccountTx.h

```diff
@@ -57,7 +57,7 @@ class AccountTxHandler
         bool validated = true;
     };
 
-    // TODO:we did not implement the "strict" field
```

Comment:
I must copied the comment from other place, account_tx's document does not mention "strict".